source/directives/checkbox-field.ts

```diff
@@ -49,6 +49,8 @@ module formFor {
      */
     model:BindableFieldWrapper;
 
+    attributes:any;
```

Comment:
**Edit** This was my only suggested change so I will just add the comment myself after merging.